### PR TITLE
user/mise: new package

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: user/mise/template.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "pkgconf",
+]
+makedepends = [
+    "libgit2-devel",
+    "openssl-devel",
+    "rust-std",
+    "zstd-devel",
+]
+checkdepends = ["bash"]
+pkgdesc = "Dev tools, env vars, task runner"
```
Comment (subnut) [inline user/mise/template.py:16]: Wouldn't "Development environment setup tool" sound better?

https://mise.jdx.dev/about.html

File: user/mise/template.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://mise.jdx.dev"
+source = f"https://github.com/jdx/mise/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "0763599841970d9a60630fbd97301d7a06d9ce5c61d9da657fda432f2204cfae"
+
```
Comment (subnut) [inline user/mise/template.py:22]: Seems to build fine and pass tests with 
```python
hardening = ["vis", "cfi"]
```

Comment (ttyyls) [reply user/mise/template.py:22]: those are ignored in the cargo build style

File: user/mise/template.py (hunk 1 above)
Comment (ttyyls) [reply user/mise/template.py:16]: sounds good

**Comment by q66** [comment]
this needs to be changed to use system lua (`--no-default-features --features=native-tls` at least plus a makedep and maybe other stuff idk)

**Pull request merged**